ui/src/app/components/direct-connection/direct-connection.component.ts

```diff
@@ -75,55 +75,63 @@ export class DirectConnectionComponent implements OnInit {
   }
 
   testConn() {
-    this.clickEvent.openDatabaseLoader('test-connection', this.connectForm.value.dbName)
+    if (typeof this.connectForm.value.dbName === "string") {
+      this.clickEvent.openDatabaseLoader('test-connection', this.connectForm.value.dbName)
+    }
```

Comment:
Very interesting, what is the error if we don't do the `if` check? Can't we simply force a cast to string for `dbName` to avoid the `if` check?

Reply:
Removed the if check and added ! - non null assertion operator